diff --git a/modules/talkinghead.mjs b/modules/talkinghead.mjs
index 441ddff6b01d05093a244441f77c60ffead695e8..08198c314fff29d4af3cb809ffeebe7959519d7e 100644
--- a/modules/talkinghead.mjs
+++ b/modules/talkinghead.mjs
@@ -2750,7 +2750,7 @@ class TalkingHead {
     if ( !this.lipsync.hasOwnProperty(lang) ) {
       const moduleName = path + 'lipsync-' + lang.toLowerCase() + '.mjs';
       const className = 'Lipsync' + lang.charAt(0).toUpperCase() + lang.slice(1);
-      import(moduleName).then( module => {
+      import(/* webpackIgnore: true */ /* turbopackIgnore: true */ moduleName).then( module => {
         this.lipsync[lang] = new module[className];
       });
     }
